frontend/src/containers/Ledger.js

```diff
@@ -214,6 +214,17 @@ export class Ledger extends Component {
   }
 }
 
+function findExpense(expenseId, expensesObj) {
+  if (expensesObj) {
+    for (let key in expensesObj) { // eslint-disable-line prefer-const
```

Comment:
for loops don't work on second iteration if you use a const there.

yes, ideally expense keyed to their id is best. That's more work than it's worth right now.